volatility3/framework/plugins/linux/mountinfo.py

```diff
@@ -258,14 +250,14 @@ def run(self):
             self.context, self.config["kernel"], filter_func=pid_filter
         )
 
-        columns = [("MNT_NS_ID", int)]
+        columns = [("MNT_NS_ID", str)]
         # The PID column does not make sense when a PID filter is not specified. In that case, the default behavior is
         # to displays the mountpoints per namespace.
         if pids:
             columns.append(("PID", int))
```

Comment:
Yeah, all the changes above from int() to str() is to take of the NotApplicable formatting to "-". Otherwise, I think I got another exception when the value is an instance of NotApplicable and the column is an int.

Reply:
Columns in the output should always accept anything derived from `BaseAbsentValue`.  If you have a situation where that goes wrong, I'd sooner get that fixed than do casting everywhere.  The point of the `BaseAbsentValue` derived classes is to allow whatever medium is display the results to display then in a way that might make more sense (so a web UI might want to highlight them in red for example).  Not being able to distinguish between an empty value and one that wasn't available for some reason will make trickier...